### DISCUSSION TITLE
Creating a looming visual that retracts

### DISCUSSION THREAD

**Discussion by professionalflyswatter** [bonsai-rx/bonsai#2044] [Q&A]
Hi I am currently working on a looming visual that needs to return back to the original state of a small circle after expanding to covering the whole screen. I am quite new to Bonsai so i am still unsure about how to implement this. This is my current workflow:
![image](https://github.com/user-attachments/assets/56a85abd-35e3-4fb7-8307-4ecd27fb52c0)

Furthermore, I have this workflow atm for the triggering of the visual using 2 cameras but now the visual keeps triggering multiple times if an object is seen in stead of just once. Is there a way to create a period in which triggering is canceled?
![image](https://github.com/user-attachments/assets/4623a597-0ec8-4219-b6b0-17e161c81272)

**Comment by professionalflyswatter** [comment]
Here are the files:
[looming visual github.zip](https://github.com/user-attachments/files/17493258/looming.visual.github.zip)

**Reply by chiyu1203** [reply]
sorry I missed this message. do you still need help?
